metagpt/provider/bedrock/utils.py

```diff
@@ -1,52 +1,84 @@
-    "anthropic.claude-instant-v1": 100000,
-    "anthropic.claude-instant-v1:2:100k": 100000,
-    "anthropic.claude-v1": 100000,
-    "anthropic.claude-v2": 100000,
-    "anthropic.claude-v2:1": 200000,
-    "anthropic.claude-v2:0:18k": 18000,
-    "anthropic.claude-v2:1:200k": 200000,
-    "anthropic.claude-3-sonnet-20240229-v1:0": 200000,
-    "anthropic.claude-3-sonnet-20240229-v1:0:28k": 28000,
-    "anthropic.claude-3-sonnet-20240229-v1:0:200k": 200000,
-    "anthropic.claude-3-haiku-20240307-v1:0": 200000,
-    "anthropic.claude-3-5-sonnet-20240620-v1:0": 200000,
-    "anthropic.claude-3-haiku-20240307-v1:0:48k": 48000,
-    "anthropic.claude-3-haiku-20240307-v1:0:200k": 200000,
-    # currently (2024-4-29) only available at US West (Oregon) AWS Region.
-    "anthropic.claude-3-opus-20240229-v1:0": 200000,
-    "cohere.command-text-v14": 4000,
-    "cohere.command-text-v14:7:4k": 4000,
-    "cohere.command-light-text-v14": 4000,
-    "cohere.command-light-text-v14:7:4k": 4000,
-    "meta.llama2-13b-chat-v1:0:4k": 4000,
-    "meta.llama2-13b-chat-v1": 2000,
-    "meta.llama2-70b-v1": 4000,
-    "meta.llama2-70b-v1:0:4k": 4000,
-    "meta.llama2-70b-chat-v1": 2000,
-    "meta.llama2-70b-chat-v1:0:4k": 2000,
-    "meta.llama3-8b-instruct-v1:0": 2000,
-    "meta.llama3-70b-instruct-v1:0": 2000,
-    "mistral.mistral-7b-instruct-v0:2": 32000,
-    "mistral.mixtral-8x7b-instruct-v0:1": 32000,
-    "mistral.mistral-large-2402-v1:0": 32000,
+    # Claude Instant v1
+    "anthropic.claude-instant-v1": 4096,
+    "anthropic.claude-instant-v1:2:100k": 4096,
+    # Claude v2
+    "anthropic.claude-v2": 4096,
+    "anthropic.claude-v2:0:18k": 4096,
+    "anthropic.claude-v2:0:100k": 4096,
+    # Claude v2.1
+    "anthropic.claude-v2:1": 4096,
+    "anthropic.claude-v2:1:18k": 4096,
+    "anthropic.claude-v2:1:200k": 4096,
+    # Claude 3 Sonnet
+    "anthropic.claude-3-sonnet-20240229-v1:0": 4096,
+    "anthropic.claude-3-sonnet-20240229-v1:0:28k": 4096,
+    "anthropic.claude-3-sonnet-20240229-v1:0:200k": 4096,
+    # Claude 3 Haiku
+    "anthropic.claude-3-haiku-20240307-v1:0": 4096,
+    "anthropic.claude-3-haiku-20240307-v1:0:48k": 4096,
+    "anthropic.claude-3-haiku-20240307-v1:0:200k": 4096,
+    # Claude 3 Opus
+    "anthropic.claude-3-opus-20240229-v1:0": 4096,
+    # Claude 3.5 Sonnet
+    "anthropic.claude-3-5-sonnet-20240620-v1:0": 8192,
+    # Command Text
+    "cohere.command-text-v14": 4096,
+    "cohere.command-text-v14:7:4k": 4096,
+    # Command Light Text
+    "cohere.command-light-text-v14": 4096,
+    "cohere.command-light-text-v14:7:4k": 4096,
+    # Command R
+    "cohere.command-r-v1:0": 4096,
+    # Command R+
+    "cohere.command-r-plus-v1:0": 4096,
+    # Llama 2 (--> Llama 3/3.1) !!!
+    # + Legacy: 2024-05-12
+    # + EOL: 2024-10-30
+    "meta.llama2-13b-chat-v1": 2048,
+    "meta.llama2-13b-chat-v1:0:4k": 2048,
+    "meta.llama2-70b-v1": 2048,
+    "meta.llama2-70b-v1:0:4k": 2048,
+    "meta.llama2-70b-chat-v1": 2048,
```

Comment:
Due to larger model can produce better result, maybe we don't need all these models, maybe keep some MoE models or size large than 30B?

Reply:
I'll comment out the legacy Llama 2 models (going EOL at EOM cf. [Model Lifecycle > Legacy Versions](https://docs.aws.amazon.com/bedrock/latest/userguide/model-lifecycle.html#versions-for-eol) for more information), the smaller (`<30B`) Llama 3 models and Mistral 7B Instruct. **Anything else?**